### ISSUE TITLE
Crash on implementation of generic and non-generic interfaces

### ISSUE THREAD

**Issue by Jayshua** [NaturalDocs/NaturalDocs#10]
I have a class which implements both the IList interface and the IList&lt;T&gt; interface.

```
public class MyList<T> : IList, IList<T> {
	/// <summary>This is a test function with xml documentation</summary>
	/// <param name="args" type="string">This is a test argument</param>
	/// <param name="second">This is another test argument</param>
	public static void MasterFunction(string args, int other) {
		Console.Write("Hello");
	}
}
```

When I attempt to run Natural Docs on it, I get this error:

```
Natural Docs has stopped because of the following error:

Value cannot be null.
Parameter name: input (Builder Thread 5)
```

Here's the crash file:

```
Generated on 11/14/2017 4:36:43 PM

Crash Message:

   Value cannot be null.
Parameter name: input (Builder Thread 5)
   (System.ArgumentNullException)

Stack Trace:

   ----- Builder Thread 5 -----
   at System.Text.RegularExpressions.Regex.Matches(String input)
   at CodeClear.NaturalDocs.Engine.Output.Components.HTMLComponent.BuildWrappedTitle(String title, Int32 commentTypeID, StringBuilder output)
   at CodeClear.NaturalDocs.Engine.Output.Components.HTMLClassPrototype.BuildParentClass(Parent parent)
   at CodeClear.NaturalDocs.Engine.Output.Components.HTMLClassPrototype.Build(Topic topic, Boolean isToolTip, IList`1 links, IList`1 linkTargets, StringBuilder output)
   at CodeClear.NaturalDocs.Engine.Output.Components.HTMLTopic.BuildPrototype()
   at CodeClear.NaturalDocs.Engine.Output.Components.HTMLTopic.Build(Topic topic, IList`1 links, IList`1 linkTargets, StringBuilder output, IList`1 embeddedTopics, Int32 embeddedTopicIndex, String extraClass)
   at CodeClear.NaturalDocs.Engine.Output.Components.HTMLTopicPage.Build(Accessor accessor, CancelDelegate cancelDelegate)
   at CodeClear.NaturalDocs.Engine.Output.Builders.HTML.BuildClassFile(Int32 classID, Accessor accessor, CancelDelegate cancelDelegate)
   at CodeClear.NaturalDocs.Engine.Output.Builders.HTML.WorkOnUpdatingOutput(CancelDelegate cancelDelegate)
   at CodeClear.NaturalDocs.Engine.Output.Manager.WorkOnUpdatingOutput(CancelDelegate cancelDelegate)
   at CodeClear.NaturalDocs.Engine.Thread.InternalStart()
   ----- Parent Thread -----
   at CodeClear.NaturalDocs.CLI.Application.Multithread(String threadName, CancellableTask task)
   at CodeClear.NaturalDocs.CLI.Application.BuildDocumentation(ErrorList errorList)
   at CodeClear.NaturalDocs.CLI.Application.Main(String[] commandLine)

Command Line:

   naturaldocs  config

Version: 2.0.1
Platform: Microsoft Windows NT 6.2.9200.0 (Win32NT)
SQLite: 3.13.0
```

**Comment by fr34kyn01535** [comment]
Ran into the same issue today. Any news?

**Comment by NaturalDocs** [comment]
Sorry, this one slipped by me somehow.  I reproduced the issue, still looking into at what part of the process this makes it choke.

**Comment by NaturalDocs** [comment]
Try this version.  These changes will be included in the next full release.

http://www.naturaldocs.org/temp/Natural_Docs_temp7.zip

**Comment by NaturalDocs** [comment]
Closing.  The changes were included in 2.0.2.
